doc/source/reference/offset_frequency.rst

```diff
@@ -26,8 +26,6 @@ Properties
     DateOffset.normalize
     DateOffset.rule_code
     DateOffset.n
-    DateOffset.is_month_start
-    DateOffset.is_month_end
```

Comment:
When browsing the preview, I notice these two methods are placed in both Properties and Methods sections.
Seems to be a mistake, so remove them here.